ModuleConfig.cfc

```diff
@@ -1,22 +1,31 @@
 component {
 
-    this.name = "cfmigrations";
-    this.author = "Eric Peterson";
-    this.description = "Keep track and run your database migrations with CFML";
-    this.version = "0.0.0";
-    this.cfmapping = "cfmigrations";
-    this.autoMapModels = false;
-    this.dependencies = [ "qb" ];
-
-    function configure() {
-        settings = {
-            migrationsDirectory = "/resources/database/migrations",
-            defaultGrammar = "BaseGrammar"
-        };
-
-        binder.map( "MigrationService@cfmigrations" )
-            .to( "#moduleMapping#.models.MigrationService" )
-            .initArg( name = "defaultGrammar", ref = "#settings.defaultGrammar#@qb" );
-    }
+	this.name          = "cfmigrations";
+	this.author        = "Eric Peterson";
+	this.description   = "Keep track and run your database migrations with CFML";
+	this.version       = "0.0.0";
+	this.cfmapping     = "cfmigrations";
+	this.autoMapModels = false;
+	this.dependencies  = [ "qb" ];
+
+	function configure() {
+		settings = {
```

Comment:
I think we need to take a `Protocol` approach here and allow users to register multiple migration sites.  For instance:
```cfc
moduleSettings = {
    "cfmigrations": {
        "managers": {
            "db1": {
                "protocol": "cfmigrations.models.managers.QBManager",
                "properties": {
                    "directory": "/resources/database/db1/migrations",
                    "baseGrammar": "MySQLGrammar@qb",
                    "datasource": "db1",
                    "useTransactions": false
                }
            },
            "db2": {
                "protocol": "cfmigrations.models.managers.QBManager",
                "properties": {
                    "directory": "/resources/database/db2/migrations",
                    "baseGrammar": "PostgresGrammar@qb",
                    "datasource": "db2",
                    "useTransactions": true
                }
            },
            "elasticsearch7": {
                "protocol": "cbElasticSearch.models.manager.ElasticMigrationsManager",
                "properties": {
                    "directory": "/resources/elasticsearch/migrations"
                }
            }
        }
    }
};
```

Then it can be grabbed from the `MigrationService`:

```cfc
variables.migrationService.getManager( "elasticsearch7" ).runAllMigrations();
```

Or even maybe a custom WireBox DSL:

```cfc
getInstance( "migrationService:db2" ).runAllMigrations();
```

Thoughts?

Reply:
I totally agree.  It prevents you from having to use the binder to manually register separate instances.  I think the DSL would be easier to manage, because with the former you would have to move all of the file-based inspection to the managers and that's agnostic code that doesn't need to be duplicated between managers.

I prefer the following DSL, myself `getInstance( "db2MigrationService@cfmigrations" )` but that's just me.

We could maintain backward compatibility for the old migration service, as well and still introduce protocols.

Reply:
I've added support for this in commit #352642c